code/game/machinery/seed_extractor.dm

```diff
@@ -60,7 +84,7 @@ obj/machinery/seed_extractor/attackby(var/obj/item/O as obj, var/mob/user as mob
 		S.use(1)
 		new /obj/item/seeds/grassseed(loc)
 
-	if(O)
+	if(O) //not sure what this portion of code is supposed to do - needs some commenting

```

Comment:
Kinda meant the rest of that process, I can't tell what it does at a glance or at a frowning stare